
This patch changes the root filesystem location and adds the video output
resolution.

Signed-off-by: Joao Pinto <jpinto@synopsys.com>
---
 include/configs/vexpress_aemv8a.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/include/configs/vexpress_aemv8a.h b/include/configs/vexpress_aemv8a.h
index 133041b..0b06f0b 100644
--- a/include/configs/vexpress_aemv8a.h
+++ b/include/configs/vexpress_aemv8a.h
@@ -200,12 +200,13 @@
 
 /* Assume we boot with root on the first partition of a USB stick */
 #define CONFIG_BOOTARGS		"console=ttyAMA0,115200n8 " \
-				"root=/dev/sda2 rw " \
+				"root=/dev/sda1 rw " \
 				"rootwait "\
 				"earlyprintk=pl011,0x7ff80000 debug "\
 				"user_debug=31 "\
 				"androidboot.hardware=juno "\
-				"loglevel=9"
+				"loglevel=9" \
+				"video=HDMI-A-1:1280x720@60" \
 
 /* Copy the kernel and FDT to DRAM memory and boot */
 #define CONFIG_BOOTCOMMAND	"afs load ${kernel_name} ${kernel_addr} ; " \
-- 
1.8.1.5
